blog/_posts/2025-05-29-azure-monitor-grafana-dashboards-portal.md

```diff
@@ -0,0 +1,141 @@
+
+We are thrilled to announce that Azure Kubernetes Service (AKS) now offers native Grafana dashboards within the Azure portal at no additional cost. This integration eliminates the complexity of maintaining separate visualization tools while delivering Grafana's powerful capabilities directly where you need them. Metrics from Container Insights, the Kubernetes metrics server, and any configured Azure Managed Prometheus endpoints are available out-of-the-box, providing comprehensive cluster observability.
+
+To get started, navigate to your AKS cluster in the Azure portal and select **Monitoring** > **Dashboards with Grafana (preview)**. You will be presented with prebuilt dashboards for cluster health, node utilization, and pod performance. From there, you may edit and add  panels, configure template variables scoped to namespaces or node pools, and save custom dashboards - all within the familiar AKS management experience. Because no Grafana server needs to be provisioned or maintained, teams can quickly adopt and customize dashboards within the AKS portal- reducing setup time, operational complexity, and accelerating access to actionable insights for SRE and DevOps workflows.
+
+![Cluster Dashboard in AKS](/assets/images/azure-monitor-grafana-dashboards/sample-grafana-dashboard.png)
+*Figure 1: Comprehensive AKS cluster dashboard showing resource utilization, performance metrics, and health status directly within the Azure portal.*
+
+### Why Grafana in Azure Portal?
+
+Grafana is celebrated for its rich panel types, templating engine, and client-side data transformations. Embedding it natively in Azure Portal offers:
+
+- Unified experience: No extra authentication or network configuration—just use your Azure login.
+- Single-pane observability: Combine Azure Metrics, Logs, and Application Insights data alongside and other Azure data sources supported by Grafana.
+- Rapid onboarding: Spin up dashboards in minutes using familiar Azure workflows and templates. Many prebuilt dashboards are available out of the box and any community dashboard using Prometheus and/or Azure Monitor can be imported.
+
+These capabilities mean faster troubleshooting, deeper insights, and a more consistent observability platform for Azure-centric workloads.
+
+### When to upgrade to Azure Managed Grafana?
+
+While Dashboards with Grafana in the Azure portal cover most common visualization scenarios, Azure Managed Grafana remains the right choice for advanced use cases, including:
+
+- Extended plugin support (including open-source and community plugins).
+- Advanced authentication, provisioning APIs, and fine-grained access control.
+- Multi-cloud and hybrid data source connectivity.
+
+**When to choose native dashboards:** Quick visibility into your Azure telemetry, minimal setup, with no additional costs.  
+**When to choose Azure Managed Grafana:** Large teams with complex governance, open-source or Enterprise, or multi-cloud data sources.
```

Comment:
as a user one of the questions on my mind when reading through Azure portal Grafana vs full AMG was cost. for the former, is there additional price as is the case with AMG?